src/firmware/app/control/trajectory_planner.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+#include "firmware/shared/math/polynomial_2d.h"
+#include "firmware/shared/math/vector_2d.h"
+
```

Comment:
All the functions here should be prefixed with `app_trajectory_planner_` to be consistent with the rest of the code in `app`, and also because it really helps track where functions are coming from.

Reply:
Added the prefix